portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,18 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds random fact about me to the page
+ */
+function addRandomFact() {
+  const facts =
+    ["I'm a fun guy", 'Obviously I love web development', "It's just more questions you have to ask me in order for me to tell you more about myself", "I just can't give you the whole spill"];
```

Comment:
I think a block-like array definition works better here: https://google.github.io/styleguide/jsguide.html#formatting-array-literals

Reply:
Okay, I will adjust it, thanks for pointing that out.